### DISCUSSION TITLE
神網是什麼?

### DISCUSSION THREAD

**Discussion by starm0113456** [share121/inter-knot#1798] [其他]
有人可以告訴我什麼是神網嗎?

**Comment by Dave-12138** [comment]
神网是一种基于魔网理论与神术术式的传输网络，它允许将神术力存储并传输，从而便于非神官用户在一定条件下使用神术。
神网的出现显著推进了魔导工业的革新，同时解决了数千种医疗难题，极大改善了人们的生活环境。

**Reply by sfhjhsdfg** [reply]
~~怎么看起来像是黎明之剑里的设定~~
